Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -423,6 +423,16 @@
         "Icon": "/SamplePages/Connected Animations/ConnectedAnimations.png",
         "BadgeUpdateVersionRequired": "Creators Update required",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/animations/ConnectedAnimations.md"
+      },
+      {
```

Comment:
Can we move this to the `Helpers` section? thanks!

Reply:
Done.